benchmarks/src/test/java/org/apache/druid/server/coordinator/BalancerStrategyBenchmark.java

```diff
@@ -77,7 +77,7 @@
   private int reservoirSize = 1;
   private double percentOfSegmentsToConsider = 100;
   private final BalancerStrategy balancerStrategy = new CostBalancerStrategy(
-      MoreExecutors.listeningDecorator(Executors.newFixedThreadPool(1))
+      MoreExecutors.listeningDecorator(Execs.multiThreaded(1, "Test-%d"))
```

Comment:
stylistic nit: Can we provide betters names than test. 
May be cost_balancer_test_%d everywhere.

Reply:
Would cost balancer be a generic enough name for all the tests? There are other tests like JettyTest

Reply:
Each test would be something specific. If its a general executor then we can go ahead with "Test-%s"